package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@adminjs/typeorm",
-  "version": "2.0.4",
+  "version": "3.0.0",
```

Comment:
Please revert this line, we use https://github.com/semantic-release/semantic-release for automatic versioning and it will set the version to 3.0.0 as long as there is a `BREAKING CHANGE: <...>` footer in the commit

Reply:
will do changing back